### DISCUSSION TITLE
How much time does it take to get github copilot ?

### DISCUSSION THREAD

**Discussion by Aryan-banafal** [community/community#16990] [Copilot Conversations] [answered]
How much time does it take to get github copilot ?
Like when will the waitlist get cleared ?
Excited to use it Hoping to get the access soon ........

**Comment by MMate2007** [comment]
I got it pretty quick, in about 1 week some months ago.

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
